modules/olm-installing-po-after.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * operators/admin/olm-managing-po.adoc
+
+:_content-type: PROCEDURE
+[id="olm-installing-po-after_{context}"]
+= Installing platform Operators after cluster creation
+
+As a cluster administrator, you can install platform Operators after cluster creation on clusters that have enabled the `TechPreviewNoUpgrades` feature set by using the cluster-wide `PlatformOperator` API.
+
+.Prerequisites
+
+- Access to an {product-title} cluster using an account with `cluster-admin` permissions.
+- Enable the `TechPreviewNoUpgrades` feature set on the cluster. See _Additional resources_ for details.
+- Install the `oc` command to your local system.
```

Comment:
Is there a way to make these parallel in construction? (might not be possible without being too convoluted)

Reply:
Was trying to match the styling from another assembly but, this has clearly gotten awkward so I will abandon that and have now made them parallel (also moved them up to the assembly-level since they were just being repeated).